### ISSUE TITLE
Intake Meeting: PHB Ranking Refactor

### ISSUE THREAD

**Issue by amenity** [cityofaustin/atd-data-tech#2823]
### Objective 
Meet with Renée to discuss what [enhancements can be made to AMD Data Tracker](https://github.com/cityofaustin/atd-data-tech/issues/2785) to support [data-informed PHB Ranking](https://github.com/cityofaustin/atd-data-tech/issues/1182). 

### Participants
Diana, Renée, Amenity — Surbhi optional

### Recording
[Microsoft Stream](https://web.microsoftstream.com/video/71e5a448-a712-4824-9233-d256ed6d16e0) (52 minutes)

------
- [x] Schedule meeting
- [x] Meet and take notes
- [x] Create resulting issues

**Comment by amenity** [comment]
## Diana's Notes
### Traffic Signal Request page (keep page)
- 311 based signal and phb requests page
- Lose/Remove `PHB Evaluations` and `PHB Studies` on this page
- Keep `PHB Evaluation` records, don't need to edit or frequently access

**FUTURE PROCESS**
- PHB Scoring isn't going to be part of Data Tracker
- Will work with Signal Engineers, Council Districts to review
  - Come up with highest ranking street segments based on GIS Data

### Signal Studies (new tracking page)
**PROPOSED IDEA**
- Create a location 
- On location details
  - Add New Signal Study a this location
  - Creates New Signal Study record 
    - Reference Renee's spreadsheet (which has many fields from the `signal_request_signal_study`
    - Create new `signal_studies` object table

### CURRENT TABLE ORGANIZATION
Locations>Signal Requests
Signals>Signal Requests
![image](https://user-images.githubusercontent.com/21112975/83304162-fc313780-a1c3-11ea-8478-715496481d58.png)
- Difference between `Signal Studies` and `PHB Studies` - they can exist on the same table? No, that's not how it is currently. 

Would want to switch out `Signal Requests` to `Studies` or `Locations`. 

### FUTURE TABLE ORGANIZATION
Locations>Studies
Studies>Traffic Studies
Studies>PHB Studies

### Migration Effort
- We will probably have to export the `signal_request_signal_studies` and the signal_request_phb_studies` table, manipulate in excel. 
- Reimport to created `Studies` table
- Export `Studies` table
- Update `signal_studies` table with connected ID to Studies table
- Update `phb_studies` table with connected ID to Studies table
- Import  `signal_studies` to create new `studies_signal_studies` table
- Import  `signal_studies` to create new `studies_phb_studies` table

**Comment by amenity** [comment]
tl;dr PHB priority will now be data-driven rather than request-driven*. Since studies will occur outside requested locations, Locations should be the parents of Studies rather than Requests. 

*We _will_ still be taking requests. 


On [Signal Request Details page](https://data.mobility.austin.gov/data-tracker/#signal-requests/signal-request-details/581eb3f24648a8b32a1faf86/):
- Remove "PHB Evaluation"/"Traffic Signal Evaluation"
- Remove "Remove PHB Studies"/"Traffic Signal Studies"

On [Data Tracker landing page](https://data.mobility.austin.gov/data-tracker/#): 
- Under programs, add "Signal Study"
<img width="594" alt="Study-List" src="https://user-images.githubusercontent.com/1463708/83914310-9f3eff80-a736-11ea-8107-c46947168307.png">

Currently have
- 152 traffic signal studies 
- 208 PHB signal studies
